csv_schema/views.py

```diff
@@ -100,8 +104,30 @@ class DataElementList(NCDRDisplay, ListView):
     paginate_by = 50
 
     def get_queryset(self, *args, **kwargs):
-        qs = super().get_queryset()
-        return qs.viewable(self.request.user)
+        qs = super().get_queryset().viewable(self.request.user)
+
+        symbol = self.request.GET.get("letter")
+
+        if not symbol:
```

Comment:
@ghickman If no symbol we should redirect to A maybe?

Reply:
If there's no `symbol` at this stage it means the URI is in the form `?key=`.  [RFC 3986](https://tools.ietf.org/html/rfc3986#section-3.4) doesn't define what to do in this situation but to me that says "a letter has not been picked".

This situation should only occur unexpectedly (user has edited the URL or we did something wrong when building a link) and we should always produce something sensible from an unexpected situation.  Since we already have the concept of an unfiltered view (no query string or an empty one) and it's also the default view I think it makes sense to reuse that here.  That way a user hits the unexpected state and is shown the default view rather than another selection which would likely be confusing.

Reply:
That's why I _think_ I used a url parameter, then redirect to A with a redirect view if you were to navigate to the direct list page.

Also google could take us to the url possibly.

I guess it doesn't matter its paginated anyway.

Reply:
Yea, I think it's best to keep the unfiltered view available and let the user pick what they want to filter by, if anything.